ovn-org/ovn-kubernetes
# Changes Metrics for IC

**- What this PR does and why is it needed**
This PR changes the subsystem from `master` to `ovnkube-controller`. Credit to @martinkennelly who had started this work initially through https://github.com/martinkennelly/ovn-kubernetes-1/commit/0ca7ed7d81820c248c1709e2e8562044a54a3aef#diff-cf3d012028107d0065b038f3ddf3f2fd82b14518b09aa7193b45bb5ef7eefebf 

**- Special notes for reviewers**
Meeting with @martinkennelly to understand why I need to move some of them into vectors with node labels when from what I can see currently for the node side metrics we don't do anything like that..

EDIT: We decided we don't need to really change any of the labels since the way we scrape things d/s will ensure the updates happen correctly.

**- How to verify it**
Trial and Error folks! :) We unfortunately don't have strong e2e CI signal for metrics upstream. So when @bpickard22 tries this out for alerts and we hit issues downstream, we will come back to fix them as bugs :D
EDIT: I am in the middle of trying out this PR d/s via https://github.com/openshift/ovn-kubernetes/pull/1727 -> let's see how that goes.

**- Description for the changelog**
`Changes metrics subsystem for ovnkbue-controller`
